src/main/java/uk/gov/ons/census/casesvc/model/dto/PayloadDTO.java

```diff
@@ -20,11 +20,11 @@
   private FieldCaseSelected fieldCaseSelected;
   private FulfilmentInformation fulfilmentInformation;
   private Metadata metadata;
+  private NewAddress newAddress;
```

Comment:
why is this called `newAddress` and not `newAddressReported` like before? Did the spec change or was it wrong before?

Reply:
Good question, discussed with Hugh and we went with NewAddess, I'd called it NewAddressReported, however in the event Dictionary although the event as a whole is called NewAddressReported, the Payload field is newAddress. 

I'm guessing that it's reusable for AddressModified perhaps.

Reply:
In the event Dictionary the Event is called NewAddressReported but the field NewAddress is where the actual payload data will be.  I was confused by this.

Reply:
Go with the dictionary